sample_apps/simple_app_python/example_app.py

```diff
@@ -0,0 +1,318 @@
+#!/usr/bin/env python3
+################################################################################
+# main.py
+################################################################################
+"""
+Python simple-app to demonstrate use of Certifier Framework APIs.
+"""
+import sys
+import os
+import argparse
+from inspect import currentframe
+
+import policy_key
+import certifier_framework as cfm
+
+###############################################################################
+# Global Variables: Used in multiple places. List here for documentation
+###############################################################################
+
+# Resolves to: <Certifier-Root>/sample_apps/simple_app_python
+THIS_SCRIPT_DIR = os.path.dirname(os.path.realpath(__file__))
+
+# Script defaults
+ATTEST_KEY_FILE                 = 'attest_key_file.bin'
+APP_DATA_DIR                    = './app1_data/'
+EXAMPLE_MEASUREMENT             = 'example_app.measurement'
+APP_OP_TYPES                    = 'cold-init, get-certified, run-app-as-client, run-app-as-server'
+PLATFORM_ATTEST_ENDORSEMENT     = 'platform_attest_endorsement.bin'
+PLATFORM_CERT_FILE              = 'platform_file.bin'
+POLICY_STORE                    = 'store.bin'
+POLICY_HOST                     = 'localhost'
+POLICY_HOST_PORT                = 8123
+SERVER_APP_HOST                 = 'localhost'
+SERVER_APP_PORT                 = 8124
+
+###############################################################################
+# main() driver
+###############################################################################
+def main():
+    """
+    Shell to call do_main() with command-line arguments.
+    """
+    do_main(sys.argv[1:])
+
+###############################################################################
+# pylint: disable=too-many-locals
+# pylint: disable=too-many-statements
+# pylint: disable=too-many-branches
+def do_main(args) -> bool:
+    """
+    Main driver for parsing arguments and run the simple-app client / server
+    components.
+    """
+    parsed_args = parseargs(args)
+
+    attest_key_file         = parsed_args.attest_key_file
+    app1_data_dir           = parsed_args.app1_data_dir.rstrip('/')
+    measurement             = parsed_args.measurement
+    operation               = parsed_args.operation
+    pf_attest_endorsement   = parsed_args.pf_attest_endorsement
+    pf_file_name            = parsed_args.pf_file_name
+    policy_host             = parsed_args.policy_host
+    policy_port             = parsed_args.policy_port
+    policy_store_file       = parsed_args.policy_store_file
+    server_app_host         = parsed_args.server_app_host
+    server_app_port         = parsed_args.server_app_port
+    print_all               = parsed_args.print_all
+
+    if print_all:
+        print('INITIALIZED_CERT_SIZE = ', policy_key.INITIALIZED_CERT_SIZE)
```

Comment:
@rgerganov -- From your comment in the draft PR that this variable is not needed -- I agree.

In code, I do use `len()` when needed. 

I felt it's just a small enough change that keeps the output in the .py and .cc file be consistent. 

Hence, I retained the generation of this `INITIALIZED_CERT_SIZE` variable.